tests/behaviour/util/mod.rs

```diff
@@ -66,7 +68,13 @@ pub async fn match_answer_concept(context: &Context, answer_identifier: &str, an
     match identifiers[0] {
         "key" => key_values_equal(context, identifiers[1], answer).await,
         "label" => labels_equal(identifiers[1], answer),
-        "value" => values_equal(identifiers[1], answer),
+        "value" => {
+            if identifiers[1].contains("2009") {
```

Comment:
@shiladitya-mukherjee  what is this?

Reply:
OOPS, sorry dont know how I overlooked it. Was just debg statement, Fixed.